tests/Doctrine/Sniffs/Classes/ExceptionClassNamingSniffTest.php

```diff
@@ -1,59 +1,63 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Doctrine\Sniffs\Classes;
 
 use SlevomatCodingStandard\Sniffs\TestCase;
 
 class ExceptionClassNamingSniffTest extends TestCase
 {
-    private const PATH_TO_CLASSES = __DIR__.'/../../../test/exception-class/';
+    private const PATH_TO_CLASSES = __DIR__ . '/../../../test/exception-class/';
 
     /**
      * @dataProvider provideValidInterfaceFiles
-     *
-     * @param string $filePath
      */
-    public function testValidClass(string $filePath)
+    public function testValidClass(string $filePath) : void
     {
         $phpcsFile = self::checkFile($filePath);
         self::assertNoSniffErrorInFile($phpcsFile);
     }
 
     /**
      * @dataProvider provideInvalidInterfaceFiles
-     *
-     * @param string $filePath
      */
-    public function testInvalidClass(string $filePath, int $line)
+    public function testInvalidClass(string $filePath, int $line) : void
     {
         $phpcsFile = self::checkFile($filePath);
 
         self::assertSame(1, $phpcsFile->getErrorCount());
         self::assertSniffError($phpcsFile, $line, 'NotAnExceptionClass');
     }
 
+    /**
+     * @return string[][]
+     */
     public function provideValidInterfaceFiles() : array
     {
         return [
-            'Extends exception' => [self::PATH_TO_CLASSES.'ValidA.php'],
-            'Implements exception' => [self::PATH_TO_CLASSES.'ValidB.php'],
-            'Extends fqcn exception' => [self::PATH_TO_CLASSES.'ValidC.php'],
-            'Implements fqcn exception' => [self::PATH_TO_CLASSES.'ValidD.php'],
-            'Implements Throwable' => [self::PATH_TO_CLASSES.'ValidE.php'],
-            'No exception' => [self::PATH_TO_CLASSES.'ValidF.php'],
-            'Abstract extends exception' => [self::PATH_TO_CLASSES.'ValidAException.php'],
-            'Abstract implements exception' => [self::PATH_TO_CLASSES.'ValidBException.php'],
+            'Extends exception' => [self::PATH_TO_CLASSES . 'ValidA.php'],
+            'Implements exception' => [self::PATH_TO_CLASSES . 'ValidB.php'],
+            'Extends fqcn exception' => [self::PATH_TO_CLASSES . 'ValidC.php'],
+            'Implements fqcn exception' => [self::PATH_TO_CLASSES . 'ValidD.php'],
+            'Implements Throwable' => [self::PATH_TO_CLASSES . 'ValidE.php'],
+            'No exception' => [self::PATH_TO_CLASSES . 'ValidF.php'],
+            'Abstract extends exception' => [self::PATH_TO_CLASSES . 'ValidAException.php'],
+            'Abstract implements exception' => [self::PATH_TO_CLASSES . 'ValidBException.php'],
         ];
     }
 
+    /**
+     * @return string[][]|int[][]
```

Comment:
How does the correct syntax for multidimensional arrays look like? Seems like the sniff is satisfied with multiple possible formats.